README.md

```diff
@@ -6,25 +6,33 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 ![Python versions](https://img.shields.io/badge/Python%20versions->=3.6-blue)
 
-This repository provides formatting for [Snakemake][snakemake] files. It follows the design and specifications of [Black][black].
+This repository provides formatting for [Snakemake][snakemake] files. It follows the
+design and specifications of [Black][black].
 
-> **⚠️WARNING⚠️**: As this project is still in the very early stages of development, and thus not stable, we strongly recommend ensuring your files are under version control before doing any formatting. Alternatively, you can pipe the file in from stdin, which will print it to the screen, or use the `--diff` option. See [Usage](#usage) for more details.
+> **⚠️WARNING⚠️**: As this project is still in the very early stages of development, and
+> thus not stable, we strongly recommend ensuring your files are under version control
+> before doing any formatting. Alternatively, you can pipe the file in from stdin, which
+> will print it to the screen, or use the `--diff` option. See [Usage](#usage) for more
+> details.
 
```

Comment:
Given we're going to make a release, I'd like to replace 'As this project is still in the very early stages of development, and thus not stable' with 'snakefmt modifies files in-place by default, thus we strongly recommend...'. We could add a sentence after that saying 'this guards against any bugs as well' to stay safe, but the jist at this point is to warn that files will get modified in place

Reply:
Yeah, agreed. I'll change this section.